conf/modules/px4io_flash.xml

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="px4io_flash">
+  <doc>
+    <description>Flashes the px4io f1 through the px4 bootloader.</description>
+  </doc>
+  <header>
+    <file name="px4io_flash.h"/>
+  </header>
+  <init fun="px4ioflash_init()"/>
+  <event fun="px4ioflash_event()"/>
+  <makefile target="ap">
+    <raw>
+        PX4IO_UART ?= UART6
```

Comment:
Since #1534 this can and should be replaced by
```
<configure name="PX4IO_UART" default="uart0" case="upper|lower"/>
<configure name="PX4IO_BAUD" default="1500000"/>

<configure name="TELEM2_UART" default="uart3" case="upper|lower"/>
<configure name="TELEM2_BAUD" default="B115200"/>
```